File: TumeKyouen/Dto/TumeKyouenModel.m
//
//  TumeKyouenModel.m
//  TumeKyouen
//
//  Created by 石倉 昇 on 2013/02/24.
//  Copyright (c) 2013年 noboru. All rights reserved.
//

#import "TumeKyouenModel.h"


@implementation TumeKyouenModel

@dynamic stageNo;
@dynamic clearDate;
@dynamic clearFlag;
@dynamic size;
@dynamic stage;
@dynamic creator;

- (NSString *)description {
    return [NSString stringWithFormat:@"stageNo = %@, size = %@, stage = %@, creator = %@, clearFlag = %@, clearDate = %@",
            self.stageNo,
            self.size,
            self.stage,
            self.creator,
            self.clearFlag,
            self.clearDate];
}

@end

File: TumeKyouen/Dto/TumeKyouenModel.swift
//
//  TumeKyouenModel.swift
//  TumeKyouen
//
//  Created by 石倉 昇 on 2016/02/01.
//  Copyright © 2016年 noboru. All rights reserved.
//

import Foundation
import CoreData

@objc(TumeKyouenModel)
class TumeKyouenModel: NSManagedObject {
    @NSManaged var stageNo: NSNumber
    @NSManaged var clearDate: NSDate
    @NSManaged var clearFlag: NSNumber
    @NSManaged var size: NSNumber
    @NSManaged var stage: String
    @NSManaged var creator: String
}
